WordPress/WordPressTest/Activity/ActivityListViewModelTests.swift

```diff
@@ -34,7 +34,7 @@ class ActivityListViewModelTests: XCTestCase {
         XCTAssertEqual(activityStoreMock.offset, 3)
     }
 
-    // Check if `loadMore` dispatchs the correct after and before date
+    // Check if `loadMore` dispatchs the correct after/before date and groups
     //
     func testloadMoreAfterBeforeDate() {
```

Comment:
```suggestion
    func testLoadMoreAfterBeforeDate() {
```

Reply:
Done in 00c9ac7bcb30977213cc2b57b5807aa616c302e7